permissions_engine/data.json

```diff
@@ -0,0 +1 @@
+{"keys": {"https://oidc:8443/auth/realms/mockrealm": "{\"keys\":[{\"kid\":\"4MNH8f81KQeKdVBD7DgCx_OxYWtolA5ccejMaxsv_JE\",\"kty\":\"RSA\",\"alg\":\"RS256\",\"use\":\"sig\",\"n\":\"hKKoHiRscyH8fqshS332Aa6tj9PkHlqgJd1bgiM2kighkIXnBTSO7r7LLcUWbYxke0gtu-UcAtqx933ESmUpqxBZ0MaDDpqSDgtbcnjdw0dmwxwHMykeDq6rn41Meg8iCFb80Eu3jL9PPMfTWpPLaRSq7U3AWUNm2-moFIXI9wFk8LH8c67RtaUz4KSIDIWFPiebUQGJM8GnMLJJE9tW6EmK6fXzow29fKATfCRSiFDhISbTi9gIyYg6rqBNRlBUev-Ap_oFkO5unb4E_fGJ2NdARMhvQ6IwHaSyGukQ7DTt-_3hJu0fNPf7L8RmmKXGD4liSkjsNGkT91UsTPK-pw\",\"e\":\"AQAB\",\"x5c\":[\"MIICoTCCAYkCBgF55zxzIzANBgkqhkiG9w0BAQsFADAUMRIwEAYDVQQDDAltb2NrcmVhbG0wHhcNMjEwNjA3MTYwNzQ5WhcNMzEwNjA3MTYwOTI5WjAUMRIwEAYDVQQDDAltb2NrcmVhbG0wggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCEoqgeJGxzIfx+qyFLffYBrq2P0+QeWqAl3VuCIzaSKCGQhecFNI7uvsstxRZtjGR7SC275RwC2rH3fcRKZSmrEFnQxoMOmpIOC1tyeN3DR2bDHAczKR4OrqufjUx6DyIIVvzQS7eMv088x9Nak8tpFKrtTcBZQ2bb6agUhcj3AWTwsfxzrtG1pTPgpIgMhYU+J5tRAYkzwacwskkT21boSYrp9fOjDb18oBN8JFKIUOEhJtOL2AjJiDquoE1GUFR6/4Cn+gWQ7m6dvgT98YnY10BEyG9DojAdpLIa6RDsNO37/eEm7R809/svxGaYpcYPiWJKSOw0aRP3VSxM8r6nAgMBAAEwDQYJKoZIhvcNAQELBQADggEBAD0UMIjA3u9IkKcYjr8whZfWTsv5en9SEspat3sH80GfVNpOCuo0hPwRmVnzt3Ll1AYZ+1a1afF1emw03wAgqNwYeyGCIYyRwl2ZfM4ZQ6I8lUgPqTUu/f1iQxPBKf3MEVMC4nZ1ISZW/B/W1fF3C+IEJcDigGPapgBsN8k/T5D/z8HuTD1+NWW8H9j2oZ+T1VRFCh28KLkPiruxQ3j9HZG/byzo6ph2aope4FickXQ2uwrqENuqg/TdaX8rDm5BtrdkK0fKbV9iVFsoGslby2Vkj3To8JMN0Po2i3OTzO4yA71a9o63NRrI3VsctOJRqRWfz7Pe6J+PIbMe0Rl5XUg=\"],\"x5t\":\"794YwtSRT6LnhHCIdvo-Cr9uNLw\",\"x5t#S256\":\"03NUB89okf9koIufLIsOKDprz8J2J6afsqsAd-o7vPg\"}]}"}}
```

Comment:
Do we want to save this file in the repo?  Not that it's bad but we'll have to generate it anyway.